### ISSUE TITLE
Uncentred Loss function

### ISSUE THREAD

**Issue by FelixBenning** [Mannheim-Probability/bayesian-descent#9]
Assuming a centred loss is unrealistic. We actually motivated a non-zero mean function for centred labels.

How would gradient descent look if we adjusted for that? 
- Issue: This mean function is not direction agnostic (therefore minimizing both the centered loss + mean function) results in a different direction than the normal gradient. This is probably some form of regularization - find the right abstractions for this to make this clean.

**Comment by FelixBenning** [comment]
Might possibly become easier with #3

**Comment by FelixBenning** [comment]
In the linear case probably: `m(w)` = `sigma ||w||^2`
